api/src/main/java/org/apache/cloudstack/api/response/CapabilitiesResponse.java

```diff
@@ -124,6 +124,10 @@ public class CapabilitiesResponse extends BaseResponse {
     @Param(description = "the retention time for Instances disks stats", since = "4.18.0")
     private Integer instancesDisksStatsRetentionTime;
 
+    @SerializedName(ApiConstants.DYNAMIC_SCALING_ENABLED)
+    @Param(description = "true if dynamically scaling for instances is enabled", since = "4.19.4")
```

Comment:
```suggestion
    @Param(description = "true if dynamically scaling for instances is enabled", since = "4.21.0")
```

@sudo87 change it if this goes in 4.21 first. cc @DaanHoogland